CalibTracker/SiStripChannelGain/interface/APVGainHelpers.h

```diff
@@ -21,13 +22,43 @@ namespace APVGain {
     std::vector<std::pair<std::string,std::string>> monHnames(std::vector<std::string>,bool,const char* tag);
 
     struct APVmon{
-        int subdetectorId;
-        int subdetectorSide;
-        int subdetectorPlane;
-        MonitorElement* monitor;
+      
+      APVmon(int v1, int v2, int v3, MonitorElement* v4) :
+	m_subdetectorId(v1),m_subdetectorSide(v2),m_subdetectorPlane(v3),m_monitor(std::move(v4)){}
```

Comment:
@mmusich for readability it might be better to put this below `public` as not everyone is aware of the default behaviour of structs

Reply:
btw. why do you use `std::move` fo a raw pointer?

Reply:
thanks, no good reason is a leftover of an experiment to move to smart pointers.